File: Resueltos_por_competidor/Jose_Caicedo/Team.cpp
/*
* Autor: BlackJose17
* Problema: Team
* Juez Online: Codeforces
* Veredicto: Accepted
* URL: https://codeforces.com/problemset/problem/231/A
**/

#include <bits/stdc++.h>
using namespace std;

int main() {
    int n;
    cin >> n;

    int count = 0;
    for (int i = 0; i < n; i++) {
        int p, v, t;
        cin >> p >> v >> t;
        if (p + v + t >= 2) count++;
    }

    cout << count << endl;
    return 0;
}

File: Resueltos_por_competidor/Juancho_Garcia/Team.java
/*
 * Autor: Juan Andres Garcia
 * Problema: Team
 * Juez online: Codeforces
 * Veredicto: Accepted
 * Url: https://codeforces.com/problemset/problem/231/A
 *  **/

import java.util.Scanner;

public class Team {

	
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		
		int n = sc.nextInt();
		
		
		int contador = 0;
		
		for(int i = 0; i < n; i++) {
			
			int manuela = sc.nextInt();
			int juanita = sc.nextInt();
			int natalia = sc.nextInt();
			
			
			
			int suma = manuela + juanita + natalia;
			
			if(suma > 1) {
				
				contador++;
			}
			
		}
		
		System.out.println(contador);
		
		
		
		
	}
}
